DiverseSelector/diversity.py

```diff
@@ -188,17 +197,9 @@ def logdet(x: np.ndarray) -> float:
     return f_logdet
 
 
-def shannon_entropy(x: np.ndarray) -> float:
+def shannon_entropy(x: np.ndarray, normalize=True, truncation=False) -> float:
```

Comment:
@FanwangM, you forgot to add these two new arguments to the `Parameters` section.

Reply:
Fixed in [6e736ec](https://github.com/theochem/DiverseSelector/pull/177/commits/6e736ecd66b848a5e94c98b3d47be52870e11c30).